build.gradle.kts

```diff
@@ -44,7 +44,8 @@ dependencies {
 
 tasks.withType<org.liquibase.gradle.LiquibaseTask>().configureEach {
     this.javaLauncher.set(project.extensions.getByType<JavaToolchainService>().launcherFor {
-        languageVersion.set(JavaLanguageVersion.of(Versions.jdk))
+        // liquibase-core 4.7.0 and liquibase-gradle 2.1.1 fails on Java >= 13
+        languageVersion.set(JavaLanguageVersion.of(11))
```

Comment:
May be change version in buildSrc?

Reply:
We still want Java 17 for our services, so this version here is and exception